api/v1/api.go

```diff
@@ -77,18 +81,160 @@ type postIsiExportResp struct {
 type IsiExport struct {
 	ID    int      `json:"ID"`
 	Paths []string `json:"paths"`
+	Clients []string `json:"clients"`
 }
 
 type getIsiExportsResp struct {
 	ExportList []*IsiExport `json:"exports"`
 }
 
+type isiThresholds struct {
```

Comment:
I'm following the pattern used in goxtremio here.  In Go, public structs start with an upper case letter and private/protected structs start with a lower case letter.